Providers/Modules/Plugins/SecurityBaseline/plugin/security_baseline_lib.rb

```diff
@@ -14,12 +14,48 @@ def initialize(log)
         def transform_and_wrap(results, host, time)
             if results.nil?
                 @log.error "Security Baseline Assessment failed; Empty input"
-                return nil, nil
+                computer = OMS::Common.get_hostname or "Unknown host"
```

Comment:
Why is there not an accompanying PR in the OMSAgent repo for [this file](https://github.com/Microsoft/OMS-Agent-for-Linux/blob/master/source/code/plugins/security_baseline_lib.rb)?